Posts/2023/06/Measuring-Download-Time.md

```diff
@@ -0,0 +1,544 @@
+---
+post_title: 'Measuring average download time'
+username: francisconabas
+categories: PowerShell
+tags: PowerShell, Automation, Performance, Measure-Command
```

Comment:
Add `post_slug` metadata. The value should be the same as the basename of the file all lowercase. If you add this, then any updates made in GitHub will update the post in WordPress.
```suggestion
post_slug: measuring-download-time
tags: PowerShell, Automation, Performance, Measure-Command
```